src/transformers/utils/quantization_config.py

```diff
@@ -216,11 +210,8 @@ def __init__(
         self,
         nbits: int = 4,
         group_size: int = 64,
-        quant_zero: bool = True,
-        quant_scale: bool = False,
-        offload_meta: bool = False,
         view_as_float: bool = False,
-        axis: int = 0,
+        axis: int = 1,
```

Comment:
```suggestion
        axis: Optional[int] = None,
```

Then add in the init : 
```py
if axis is None: 
   axis=1
   logger.info("Setting axis=1 as faster backends such as torchao or bitblas are only compatible with it.")
```